zebra/zebra_vty.c

```diff
@@ -1556,8 +1476,18 @@ DEFPY (show_route,
 {
 	afi_t afi = ipv4 ? AFI_IP : AFI_IP6;
 	struct vrf *vrf;
+	struct zebra_router_table *zrt;
 	int type = 0;
 
+	if (!vrf_is_backend_netns()) {
+		if ((vrf_all || vrf_name) && (table || table_all)) {
+			if (!!json)
+				vty_out(vty, "{}\n");
+			else
+				vty_out(vty, "vrf and table can not be both set\n");
```

Comment:
Let's give a reason why this cannot both be set( give a bit more of a breadcrumb )

Reply:
lets follow this up on this thread:
                                vty_out(vty, "Linux vrf backend already points to table id\n");
                                vty_out(vty, "Either remove table parameter or vrf parameter\n");

Reply:
@donaldsharp

Reply:
Hi donald, ok for you ?

Reply:
that would be fine.